### PR TITLE
Update http_max_tries setting default

### PR DESCRIPTION
**Pull request by kssenii**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: src/Core/Settings.h [hunk 1]
```diff
@@ -547,7 +547,7 @@ class IColumn;
     M(Int64, remote_fs_read_max_backoff_ms, 10000, "Max wait time when trying to read data for remote disk", 0) \
     M(Int64, remote_fs_read_backoff_max_tries, 5, "Max attempts to read with backoff", 0) \
     \
-    M(UInt64, http_max_tries, 1, "Max attempts to read via http.", 0) \
+    M(UInt64, http_max_tries, 10, "Max attempts to read via http.", 0) \
```
Comment (alesapin) [inline src/Core/Settings.h:550]: Are you sure that we have to add retries everywhere where we use HTTP?

**Review by alesapin — COMMENTED** [review]
.

**Review by alesapin — APPROVED** [review]

**Comment by alesapin** [comment]
@mergifyuo update

**Comment by alesapin** [comment]
https://github.com/Mergifyio update

**Comment by alesapin** [comment]
@Mergifyio update

**Comment by kssenii** [comment]
@Mergifyio update

**Pull request merged**